Make 3DS guide easier to understand (#224)
PretendoNetwork/website: docs/en_US/install/3ds.md

# 3DS/2DS Family

<div class="tip">
	ℹ️ This guide assumes that you have a <b>Homebrewed System running the latest version of the Luma3DS custom firmware</b>, if you don't please follow this <a href="https://3ds.hacks.guide/" target="_blank">guide</a> on how to homebrew your system first.
</div>

You can get a 3DS onto Pretendo Network in 5 easy steps:
1. [Downloading Nimbus](#downloading-nimbus-to-your-sd-card)
2. [Enabling Luma patches](#luma-patches)
3. [Installing Nimbus to HOME Menu](#installing-nimbus-app-to-home-menu)
4. [Using Nimbus](#using-nimbus)
5. [Signing into Pretendo](#signing-into-your-pnid)

## Downloading Nimbus to your SD card

> Before you start, make sure your 3DS is turned off, and insert the SD card that you use for your Nintendo 3DS into your computer.

Reviewer: ```suggestion
Before you begin, power off your console and insert it's SD card into your computer
```

> Once you've done that, navigate to the [releases](https://github.com/PretendoNetwork/Nimbus) page on the Nimbus GitHub repository.

Reviewer: https://github.com/PretendoNetwork/Nimbus/releases/latest will always redirect to the latest release

```suggestion
Once inserted, download the latest [Nimbus release](https://github.com/PretendoNetwork/Nimbus/releases/latest).
```

<img src="/assets/images/docs/install/3ds/releases-highlight.png" width=100% height=auto/>

> There's a few different ways to use Nimbus, but for the best experience, download the `combined.[version].zip` file onto your computer. This file allows you to launch Nimbus both from your HOME Menu and the Homebrew Launcher. The other two files only do one or the other.

Reviewer: Building off the "professional wording" comment, maybe something like

```suggestion
Nimbus is available as both a 3dsx app and an installable CIA. The releases page offers downloads for both. Select the version you would like to use, or select the `combined.[version].zip` archive to use both
```

<img src="/assets/images/docs/install/3ds/zip-highlight.png" width=100% height=auto/>

> Extract the contents of the zip file you just downloaded and copy all of the contents of the zip file to the root of your SD card. If your computer asks you, merge files.

Reviewer: ```suggestion
Extract the contents of the zip archive to the root of your SD card. If you are asked to merge or overwrite files, accept the changes.
```

> You should now have the `0004013000002F02`, `0004013000003202`, and `0004013000003802` Luma patches along with the `nimbus.3dsx` and `nimbus.cia` homebrew.

Author: Yeah, I'll probably do the changes you've suggested later on, bit too early for me to do right now, it's not even 9am lol

> 2) `0004013000003202` - Friends system module. This patch replaces the `https://nasc.nintendowifi.net` URL with our servers URL 

Reviewer: I agree, maintaining this list would become annoying

3) `0004013000003802` - act system module. This patch replaces the `https://account.nintendo.net/v1/api/` URL with our servers URL

<div class="tip">
ℹ️ <b>Skip this step if Luma patches are already enabled on your console.</b>
</div>

> In order to make use of the patches listed above, you will need to enable Luma patches on your console. Hold the SELECT button on your 3DS and power it on.

Reviewer: Button prompts should be made more clear. Since we use markdown, just using `BUTTON` is enough

```suggestion
In order to make use of the patches listed above, you will need to enable Luma patches on your console. Hold the `SELECT` button on your 3DS and power it on.
```

> You should be greeted to a black screen with a few tickboxes. You'll need to tick 'enable loading external FIRMS and modules' and 'enable game patching' to ensure Pretendo works correctly. After that, press START on your console to save and boot into your 3DS.

Reviewer: ```suggestion
Ensure the following options are enabled

- `Enable loading external FIRMs and modules`
- `Enable game patching`

Press `START` to save and continue.
```

## Installing Nimbus app to HOME Menu

<div class="tip">
ℹ️ <b>Skip this step if you downloaded the 3dsx only zip file.</b>
</div>

> If you downloaded the combined or cia zip files, you can install the Nimbus application to the HOME Menu for quick and easy access. Here's how to do that.

Reviewer: ```suggestion
If you downloaded the combined or cia archives, you can install Nimbus to the HOME Menu for quick access.
```

> First of all, open up the FBI application on your 3DS HOME Menu. Once it is loaded, click on SD and then click on cias. Next, locate `nimbus.cia` and click on it. In the menu that appears, press the install and delete CIA button. This installs the application to your HOME Menu.

Reviewer: FBI also ships as a 3dsx app, so it may not be on the home menu, and not everyone may have it already

```suggestion
Open FBI. If you do not have FBI, download the latest release from [GitHub](https://github.com/Steveice10/FBI/releases/latest). Select `SD`, then `cias`. Find and select `nimbus.cia`. Select either `Install CIA` or `Install and delete CIA`
```

> Once it has finished installing, press the HOME button and exit FBI. You should see a message that a new application has been added to the HOME Menu. Click OK and you'll now have Nimbus on your HOME Menu for easy access.

Reviewer: ```suggestion
Once it has finished installing, press the HOME button and exit FBI. You should see a message that a new application has been added to the HOME Menu. Click OK and you'll now have Nimbus on your HOME Menu.
```

> ## Using Nimbus
> 
> To use Pretendo servers instead on Nintendo's servers, you'll need to open up Nimbus. If you followed the previous step, you can find it on your HOME Menu, or you can also find it in the Homebrew Launcher. Once inside Nimbus, click on Pretendo to switch to your Pretendo profile.
> 
> You can also repeat the same steps to go back to the Nintendo profile if you want to use Nintendo's servers.
> 
> Your active profile persists between reboots, so there's no need to open up Nimbus to switch to Pretendo (or to Nintendo) every time you turn on your 3DS.

Reviewer: It's already clear based on earlier information what Nimbus does, a lot of this can be simplified

```suggestion
## Using Nimbus

Depending on how you installed Nimbus, launch it either through the Homebrew Launcher or the 3DS Home MENU. Select either `Pretendo` or `Nintendo` to swap between services

Your selection persists between reboots
```

## Signing into your PNID

Once inside the Pretendo profile, you'll need to sign into your Pretendo Network ID to fully enjoy Pretendo. If you haven't already, [create a Pretendo account](/account) on your computer or phone.

On your 3DS, open up System Settings and tap on Nintendo Network ID Settings. It'll take a while to load, but once you do, follow along until the part where you get asked whether you want to link your account. Select `Yes` and use the credentials you used to create your Pretendo account. Once you've done that, continue normally and you should be all set up.

> Congratulations! You can now use Pretendo Network and enjoy 3DS games online even after April 2024.

Reviewer: There is currently nothing implemented that requires a PNID to be used on the 3DS. The 3DS does not use NNIDs for the vast majority of online play, so this whole section is pretty misleading. I would honestly suggest removing it until the time comes when we offer services which do require a PNID

Author: Yeah, to be honest I never really actually paid much attention to that tbh when playing stuff.

I still want to keep it in just because of stuff like Miiverse once that's available to the public
